src/main/docs/guide/sdkv2/s3.adoc

```diff
@@ -1,4 +1,12 @@
-To use an S3 client, add the following dependency:
+Micronaut provides a high-level, uniform object storage API that works across the major cloud providers: https://micronaut-projects.github.io/micronaut-object-storage/latest/guide/[Micronaut Object Storage].
+
+To get started, select the `object-storage-aws` feature in https://micronaut.io/launch?type=DEFAULT&name=demo&package=com.example&javaVersion=JDK_11&lang=JAVA&build=GRADLE&test=JUNIT&features=object-storage-aws&activity=preview&showing=build.gradle[Micronaut Launch], or add the following dependency:
```

Comment:
Maybe, it is better to replace the link with `https://micronaut.io/launch?features=object-storage-aws` so that we don't have to update the docs when we remove 11 baseline.

Reply:
Good point! Will do that across the other PRs too